recipe/meta.yaml

```diff
@@ -12,31 +12,30 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 3
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - toolchain
     - python
-    - numpy x.x
```

Comment:
I don't think you can do that b/c this package seems to rely on numpy's ABI. See https://github.com/refnx/refnx/blob/master/setup.py#L153

Also, without [re-rendering](https://conda-forge.org/docs/conda_smithy.html), this change is not in effect.

Last but not least the `numpy x.x` is deprecated. See https://conda-forge.org/docs/meta.html#building-against-numpy for a better way to do that.

Reply:
Thanks for these comments. Everytime I come to update the recipe it seems that the process has changed significantly, so it's a large learning process again.

Reply:
conda-forge actually evolves quite slowly if compared to similar systems :wink: 

But I get that. The only way to keep in touch is to hang out in our [gitter](https://gitter.im/conda-forge/conda-forge.github.io) channel

Reply:
PS: I am using https://conda-forge.org/docs/webservice.html to re-render this recipe for you.